api/internal/store/entity/live.go

```diff
@@ -0,0 +1,49 @@
+package entity
+
+import (
+	"time"
+
+	"github.com/and-period/furumaru/api/pkg/uuid"
+	"gorm.io/datatypes"
+	"gorm.io/gorm"
+)
+
+// ライブ情報
+type Live struct {
+	ID             string         `gorm:"primaryKey;<-:create"`           // テンプレートID
+	ScheduleID     string         `gorm:""`                               // 開催スケジュールID
+	Title          string         `gorm:""`                               // タイトル
+	Description    string         `gorm:""`                               // 説明
+	ProducerID     string         `gorm:""`                               // 生産者ID
+	StartAt        time.Time      `gorm:""`                               // 配信開始日時
+	EndAt          time.Time      `gorm:""`                               // 配信終了日時
+	Canceled       bool           `gorm:""`                               // 配信中止フラグ
+	Recommends     []string       `gorm:"default:null"`                   // おすすめ商品一覧
```

Comment:
RecommendsJSONフィールドの方がDBに保存する内容になるから、ここのフィールドタグは `-` かも

```suggestion
	Recommends     []string       `gorm:"-"`                   // おすすめ商品一覧
```